### ISSUE TITLE
Correctifs bouton 'consulter les 250 démarches' sur la PA

### ISSUE THREAD

**Issue by lucaa** [DISIC/observatoire#1134]
* il est plus grand que les autres boutons (taille plus large DSFR) sur la maquette et pas l'implémentation
* en mobile il ne doit pas prendre 100% de la largeur de l'écran

**Comment by ClemDee** [comment]
Yes you could, but then you would lose the ability to have a large button only for desktop layout. That's why I used a media query in css. If you prefer I can use the bootstrap class instead (removing the ability to have the button smaller on mobile).
